samples/client/petstore/java/default/src/main/java/io/swagger/client/model/Pet.java

```diff
@@ -6,6 +6,7 @@
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 import io.swagger.client.model.Category;
+import java.util.*;
```

Comment:
I don't think this change is required for this PR.

We'll update (remove) it in our next Java-related PR.